language/de/cli.php

```diff
@@ -89,8 +89,8 @@
 
 	'CLI_DESCRIPTION_UPDATE_CHECK'					=> 'Prüft, ob das Board auf dem neuesten Stand ist.',
 	'CLI_DESCRIPTION_UPDATE_CHECK_ARGUMENT_1'		=> 'Name der zu überprüfenden Erweiterung (bei „all“ werden alle Erweiterungen überprüft)',
-	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_CACHE'		=> 'Führt den „check“-Befehl auf Basis der im Cache gespeicherten Informationen aus.',
-	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_STABILITY'	=> 'Führt den „check“-Befehl wahlweise mit der Stabilität „stable“ oder „unstable“ aus.',
+	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_CACHE'		=> 'Führt den Befehl („check“) auf Basis der im Cache gespeicherten Informationen aus.',
```

Comment:
Naja wie gesagt steht das in der Hilfe zum Befehl, ich finde das "check" auch in Klammern überflüssig ;)

Wenn du willst kannst du es aber auch so lassen.